.travis.yml

```diff
@@ -137,9 +144,7 @@ install:
    - which python
    - python --version
    - which pip
-   - which nosetests
-   - nosetests --version
-   - nosetests --plugins
+   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then which nosetests; nosetests --version; nosetests --plugins; fi
```

Comment:
`nose` seams to not be installed: 
```
  Nosetests unavailable. Using fallback test harness.
  Fewer tests will be excuted.
```

Reply:
yes, it is intentional. Initially there was some [issue](https://travis-ci.org/lekshmideepu/nest-simulator/jobs/521134812#L347) with the nose test installation on Travis OSX image.  As @jougs is rewriting the test suite, he suggested to turn it OFF for OSX build at the moment.